backends/gstreamer/src/player.rs

```diff
@@ -394,6 +411,17 @@ impl GStreamerPlayer {
                 }
             });
 
+        let inner_clone = inner.clone();
+        inner
+            .lock()
+            .unwrap()
+            .player
+            .connect_property_rate_notify(move |_| {
+                let inner = inner_clone.lock().unwrap();
+                inner.player.set_rate(inner.rate);
```

Comment:
I don't think this is the right place to set the `rate` property. IIUC this event is triggered whenever the `rate` property changes, so we are effectively creating a loop here (unless GStreamer controls this somehow).

You are probably observing the `simple_player` test pass after this change because this event is never triggered (?).

I think the right place to set the rate is when we get the media info and we are able to check if the stream is seekable, as we were doing previously.

Reply:
Yeah, it makes sense, I didn't fully understand what was happening there but I got it now. Ok, so I modified the rate update as it was previously done.